pkg/tls/certificatemanagement/certificatebundle.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
```

Comment:
So this comment can be addressed after this is merged (let's leave the package here for now), but I wonder if we should just remove the intermediate `certificatemanagement` package. Having these components under the tls package seems correct.